tests/test_series.py

```diff
@@ -1236,3 +1236,32 @@ def cond2(x: pd.Series[int]) -> pd.Series[bool]:
 
     cond3 = pd.Series([False, True, True])
     check(assert_type(s.where(cond3, other=0), "pd.Series[int]"), pd.Series, int)
+
+
+def test_bitwise_operators() -> None:
+    s = pd.Series([1, 2, 3, 4])
+    # for issue #348 (bitwise operators on Series should support int)
+    check(assret_type(s & 3, "pd.Series[int]"), pd.Series,int)
```

Comment:
you misspelled `assert_type()` in 3 places.

You should test locally with `poetry run poe test` before committing.

Reply:
Sorry sir small mistake on my local setup I wrote it correctly but after copying here I unknowingly changed the spelling